pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.styles.ts

```diff
@@ -0,0 +1,174 @@
+import { css } from 'emotion';
+import { GrafanaTheme } from '@grafana/data';
+import { selectThemeVariant, stylesFactory } from '@grafana/ui';
+
+export const getStyles = stylesFactory((theme: GrafanaTheme) => {
+  const backgroundColor = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#161719' }, theme.type);
+  // @ts-ignore
+  const borderColor = selectThemeVariant({ light: theme.colors.gray85, dark: '#292929' }, theme.type);
+  const headerBackground = selectThemeVariant({ light: 'rgb(247, 247, 249)', dark: '#3D3D3D' }, theme.type);
+  const textColor = selectThemeVariant(
+    // @ts-ignore
+    { light: theme.colors.gray85, dark: 'rgba(255, 255, 255, 0.8)' },
+    theme.type
+  );
+
+  return {
+    /* This will make the table scrollable when it gets too small */
+    tableWrap: (size) => css`
+      display: block;
+      max-width: ${`${size.x}px` || '100%'};
+      // overflow-x: auto;
+      // overflow-y: scroll;
+      max-height: ${`${size.y}px` || 'auto'};
```

Comment:
This will always evolve to the left side. Need to first check if size.s is nullish, then proceed with interpolation.